distribution/lib/Standard/Base/0.0.0-dev/src/Math.enso

```diff
@@ -1,9 +1,7 @@
 import project.Data.Numbers.Decimal
 import project.Data.Numbers.Number
 
-## Alias Pi (Constant)
-
-   The mathematical constant pi, equal to the ratio of a circle circumference
+## The mathematical constant pi, equal to the ratio of a circle circumference
```

Comment:
Shouldn't these constants get a `GROUP`? I assume `Math` would fit just right.

Reply:
Deliberately not so they don't end up too high in the static list.

Reply:
Ah. I assume the `Constants` group would have the same problem?

Reply:
Yep Constants ends up at the top.